docs: Add external dependencies (#363)
stackabletech/hive-operator: docs/modules/hive/pages/index.adoc

= Stackable Operator for Apache Hive

This is an operator for Kubernetes that can manage https://hive.apache.org[Apache Hive] metastores.
The Apache Hive metastore (HMS) stores information on the location of tables and partitions in file and blob storages such as HDFS and S3.

Only the metastore is supported, not Hive itself.

> Due to several reasons we do not feel that running Hive on Kubernetes makes much sense these days.

Reviewer: ```suggestion
There are several reasons why running Hive on Kubernetes may not be an optimal solution.
```

> The most obvious reason being that Hive requires YARN as execution framework, which basically tries to do the same job as Kubernetes - i.e. assign resources. 

Reviewer: ```suggestion
The most obvious reason is that Hive requires YARN as an execution framework, and YARN assumes much of the same role as Kubernetes - i.e. assigning resources. 
```

> For this reason we decided to provide xref:trino:index.adoc[Trino] as a query engine in the Stackable Data Platform instead of Hive - which still uses the Hive Metastore, hence this operator exists.

Reviewer: ```suggestion
For this reason we provide xref:trino:index.adoc[Trino] as a query engine in the Stackable Data Platform instead of Hive. Trino still uses the Hive Metastore, hence the inclusion of this operator as well.
```
